src/App.tsx

```diff
@@ -1,19 +1,156 @@
 import React from 'react';
-import './App.css';
+import './App.scss';
 
 import users from './api/users';
+import todos from './api/todos';
 
-const App: React.FC = () => {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
+const copyTodos = [...todos];
 
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
-};
+export class App extends React.Component <{}, {}> {
+  state = {
+    title: '',
+    userId: 0,
+    noUser: false,
+    noTitle: false,
+  };
 
-export default App;
+  handleChangeUser = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    const { value } = event.target;
+
+    this.setState({
+      userId: Number(value),
+      noUser: false,
+    });
+  };
+
+  handleChangeTitle = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { name, value } = event.target;
+
+    this.setState({
+      [name]: value,
+      noTitle: false,
+    });
+  };
+
+  submitForm = () => {
+    const {
+      title, userId, noUser, noTitle,
+    } = this.state;
+
+    if (userId !== 0 && title !== '') {
+      copyTodos.push(
+        {
+          userId: +userId,
+          title,
+          id: copyTodos.length + 1,
+          completed: false,
+        },
+      );
+
+      this.setState({
+        title: '',
+        userId: 0,
+      });
+    } else {
+      if (!noUser) {
+        this.setState({ noUser: true });
+      }
+
+      if (!noTitle) {
+        this.setState({ noTitle: true });
+      }
+    }
+  };
+
+  render() {
+    return (
+      <div className="App">
+        <h1>Add TODO form</h1>
+        <form
+          onSubmit={(event) => {
+            event.preventDefault();
+          }}
+          action=""
+          method="post"
+          className="form"
+        >
+          <div className="wraper">
+            <select
+              name="user"
+              value={this.state.userId}
+              onChange={this.handleChangeUser}
+              className="form__user"
+            >
+              <option value="0">
+                Choose a user
+              </option>
+              {
+                users.map((user) => {
+                  return (
+                    <option
+                      value={user.id}
+                      key={user.id}
+                    >
+                      {user.name}
+                    </option>
+                  );
+                })
```

Comment:
Avoid using return in JSX.
```suggestion
                users.map((user) => (
                  <option
                    value={user.id}
                    key={user.id}
                  >
                    {user.name}
                  </option>
                )
```